tobac/segmentation.py

```diff
@@ -323,7 +324,9 @@ def watershedding_3D(track, field_in, **kwargs):
     warnings.simplefilter("default", DeprecationWarning)  # reset filter
 
     kwargs.pop("method", None)
-    return segmentation_3D(track, field_in, method="watershed", **kwargs)
+    if 'dxy' not in kwargs:
+        raise ValueError("Need grid spacing `dxy`.")
+    return segmentation(track, field_in, dxy=kwargs.get('dxy'), method="watershed", **kwargs)
 
```

Comment:
This will throw a `TypeError` because `segmentation()` gets multiple values for keyword argument `dxy` (one time defined via `dxy=kwargs.get('dxy')` and then again through `**kwargs`. Maybe this could be an alternative? 

```
if 'dxy' not in kwargs:
        raise ValueError("Need grid spacing `dxy`.")

dxy = kwargs.get("dxy")
kwargs.pop("dxy", None)    

return segmentation(track, field_in, dxy=dxy, method="watershed", **kwargs)
```
Or you have a better idea :)

Reply:
Great suggestion, and great catch!